OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,509 @@
+
+We envision a 3DM as being:
+
+*   A permissionless and open data delivery market, composed of one or more crypto-economic models
+*   A fair exchange relationship between service providers and users
+*   A cost efficient way for publishers to distribute their content
+*   A unique set of business opportunities including:
+    *   Delivering data to end users
+    *   Carrying data between multiple disconnected locations or with high latency between them (Data Muling and/or Package Switching)
+    *   Create ephemeral distribution networks for large gatherings
+    *   Power the next generation of Web 3.0 applications
+
+This new field is ripe with unique business opportunities given the growing demand from users for access to large datasets, videos and so on, the panoply of super-powered mobile devices that end users have access to and the limitations imposed by physics in ultimately moving increasingly large data at high speeds through fiber.
+
+In this Open Problem definition, we introduce  the multiple areas of research (or subOpenProblems), what we know so far for each one of them and link to some new ideas being explored.
+
+### Definition of the actors
+
+For convenience and shared understanding, we define here the agents within a 3DM network. These are:
+
+*   **Clients** - Fetching data from Providers
+*   **Providers** - Offer the service of data delivery to clients
+*   **Content Publishers** - The creators of content (or intermediaries) that want to have content distributed. There might be situations in which Content Publishers are willing to pay for the service consumed by Clients
+
+
+### Expected requirements
+
+We present this list as a guide to protocol designers of what we expect a 3DM implementation to meet. This list is non-exhaustive and presents some flexibility between MUST and SHOULD. The requirements identified are:
+
+*   **_MUST be Decentralized and not just federated_**
+    *   Anyone should be able to join and leave at any time, without requiring permission
+*   **_The exchanges of value_** **_MUST be verifiable_** 
+    *   The payment for bandwidth/latency in token should match what has been agreed and fulfilled
+    *   The payment should be fulfilled if the SLA is fulfilled
+    *   Parties should be able to verify that the service provided was correct (e.g. received the right file)
+*   **_SHOULD be Trustless_**
+    *   Ideally, the network can perform the operations without having to leverage Trust in third parties and/or the participants of the exchange
+    *   Nevertheless, Trust can, and should, be considered as many designs might benefit from an element of Trust to increase the quality of service (e.g. reputation supports relaxing constraints)
```

Comment:
```suggestion
    *   Ideally, the network can perform the operations without having to leverage trust in third parties and/or the participants of the exchange
    *   Nevertheless, designs might benefit from an element of trust to increase the quality of service (e.g. reputation mechanisms may allow for relaxed constraints)
```

Written like that, it appears "Trust" is a specific concept in the context of the system, when it's just generally defined trust.